solutions/coci-16-zamjene.mdx

```diff
@@ -0,0 +1,269 @@
+---
+id: coci-16-zamjene
+source: COCI 2016
+title: Zamjene
+author: Kevin Sheng
+---
+
+[Official Analysis (C++)](https://github.com/mostafa-saad/MyCompetitiveProgramming/blob/master/Olympiad/COCI/official/2017/contest2_solutions)
+
+## Explanation
+
+If you've done [Wormhole Sort](http://usaco.org/index.php?page=viewproblem2&cpid=992),
+this problem should look a bit familiar.
+From that problem's editorial, we know that it's possible to sort a component
```

Comment:
this way of mentioning the editorial is iffy, but it's alright. if you can, summarize a little bit.